.github/workflows/build-push-template.yml

```
@@ -0,0 +1,48 @@
+name: Build and Push template
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-push-template:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build and push the recipes Docker image
+      uses: docker/build-push-action@v1 # https://github.com/docker/build-push-action
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        dockerfile: .devcontainer/Dockerfile
+        repository: jeluard/substrate-playground-template-recipes
+        tags: latest
+        add_git_labels: true
+        tag_with_ref: true
+        tag_with_sha: true
+
+    - run: echo ::set-output name=image::jeluard/substrate-playground-template-recipes:sha-${GITHUB_SHA::7}
+      id: env
+
+    - name: Update devcontainer.json
+      run: |-
+        t=$(mktemp)
+        cat .devcontainer/devcontainer.json | jq ".image = \"${{ steps.env.outputs.image }}\"" > ${t} && mv ${t} .devcontainer/devcontainer.json
+
+    - name: Commit .devcontainer.json update
+      uses: EndBug/add-and-commit@v4
+      with:
+        message: ":bookmark: update image"
```

```suggestion
        message: ":bookmark: Update Docker Image for Substrate Playground"
```

---

Wold also be fine with
```suggestion
        message: "🔖 Update Docker Image for Substrate Playground"
```